From 2f79819d26eb481487ab338a0d211d70d79c6ac4 Mon Sep 17 00:00:00 2001
From: Vratislav Podzimek <vpodzime@redhat.com>
Date: Tue, 16 Oct 2012 08:07:05 +0000
Subject: There is no anaconda object available in writeSysconfigKernel

This change was introduced in commit c308c8978e2d27 that is a port
from rhel6-branch where we use anaconda.id.bootloader instead of
storage.bootloader.
---
diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index 58c3690..c0c12df 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -2140,7 +2140,7 @@ def writeSysconfigKernel(storage, version):
     f.write("\n")
     f.write("# DEFAULTKERNEL specifies the default kernel package type\n")
     f.write("DEFAULTKERNEL=%s\n" % kernel)
-    if anaconda.id.bootloader.trusted_boot:
+    if storage.bootloader.trusted_boot:
         f.write("# HYPERVISOR specifies the default multiboot kernel\n")
         f.write("HYPERVISOR=/boot/tboot.gz\n")
         f.write("HYPERVISOR_ARGS=logging=vga,serial,memory\n")
--
cgit v0.9.0.3
